lib/ruby-box/item.rb

```diff
@@ -20,15 +20,23 @@ def self.has_many_paginated(*keys)
       keys.each {|key| @@has_many_paginated << key.to_s}
     end
 
-    def move_to( folder_id, name=nil )
+    def move_to(folder_id)
+      url = "#{RubyBox::API_URL}/#{resource_name}/#{id}"
+      uri = URI.parse(url)
+
       # Allow either a folder_id or a folder object
       # to be passed in.
       folder_id = folder_id.id if folder_id.instance_of?(RubyBox::Folder)
+      self.parent = { "id" => folder_id }
 
-      self.name = name if name
-      self.parent = {"id" => folder_id}
+      request = Net::HTTP::Put.new(uri.path, {
```

Comment:
Just noticed this duplicates a fair bit of code from `#update`. What do you think about making `#update` more flexible and continuing to use it? Something like this:

```ruby
def move_to(folder_id_or_folder_object)
  folder_id = folder_id_or_folder_object.id if folder_id_or_folder_object.instance_of?(RubyBox::Folder)
  self.parent = { "id" => folder_id }
  update(["parent"])
end

def update(update_fields = nil)
  reload_meta unless etag

  update_fields ||= self.update_fields
  uri = URI.parse("#{RubyBox::API_URL}/#{resource_name}/#{id}")

  request = Net::HTTP::Put.new(uri.path, {
    "if-match" => etag,
    "Content-Type" => 'application/json'
  })
  request.body = JSON.dump(serialize(update_fields))

  @raw_item = @session.request(uri, request)
  
  self
end
```